string response_to
bool success
string error_message
string format
string type
int32 speed_of_sound
int32 mounting_rotation
bool acoustic_enabled
bool dark_mode
string range_mode
bool periodic_cycling_enabled
